src/book.cpp

```diff
@@ -208,22 +227,67 @@ std::string Book::getHumanReadableIdFromPath() const
   return id;
 }
 
+void Book::setFavicon(const std::string& favicon)
+{
+  getMutableDefaultIllustration().data = favicon;
+}
+
+void Book::setFaviconMimeType(const std::string& faviconMimeType)
+{
+  getMutableDefaultIllustration().mimeType = faviconMimeType;
+}
```

Comment:
We use them only in tests (so we don't really use them).
We probably don't want to keep this old API to set the illustration.
If we really want to provide such API (I'm not sure we should), it should be at least some kind of `setDefaultIllustration`.